docs/tutorials/api/datasets.ipynb

```diff
@@ -0,0 +1,877 @@
+{
```

Comment:
<p>This an aparte but I think we need a <code>MapDataset.peek</code>  method as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>